server/plugin.go

```diff
@@ -168,8 +171,9 @@ func (p *Plugin) handleTelemetry(w http.ResponseWriter, r *http.Request) {
 
 	telemetryRequest, err := GetTelemetryPayloadFromJSON(r.Body)
 	if err != nil {
-		p.API.LogError("Unable to get telemetry payload from JSON err=" + err.Error())
-		p.handleErrorWithCode(w, http.StatusBadRequest, "Unable to get telemetry payload from JSON.", err)
+		msg := "Unable to get telemetry payload from JSON"
+		p.API.LogError(msg, "err", err.Error())
+		p.handleErrorWithCode(w, http.StatusBadRequest, msg, err)
```

Comment:
Sorry I was unclear about my request before. We shouldn't edit the log statements in functions unrelated to this PR's purpose. We can clean this up in a separate effort/PR

Reply:
Sorry too. should I put everything back the way it was before?

Reply:
Yes everything that is not related to the main feature/effort of the PR

Reply:
Ideally we do change the logging back the way it was for places unrelated the purpose of the PR, but I don't want to drag this PR on for that reason